src/tools/fuzzing/fuzzing.cpp

```diff
@@ -1123,6 +1127,7 @@ Expression* TranslateToFuzzReader::_makeConcrete(Type type) {
            WeightedOption{&Self::makeBreak, Important},
            &Self::makeCall,
            &Self::makeCallIndirect)
+      .add(FeatureSet::ExceptionHandling, &Self::makeTry)
```

Comment:
I think `makeBreak` is here because there are variations of Break that can have a concrete type (`br_if` with a value), so this function makeConcrete needs to handle those. But Throw is always unreachable?

Reply:
Ah sorry, I missed this function is `_makeConcrete`...